crates/stdx/src/rand.rs

```diff
@@ -0,0 +1,21 @@
+//! We don't use `rand`, as that's too many things for us.
+//!
+//! Currently, we use oorandom instead, but it misses these two utilities.
+//! Perhaps we should switch to `fastrand`, or our own small prng, it's not like
+//! we need anything move complicatied that xor-shift.
```

Comment:
This PR is already merged but it looks like there are at least 3 typos in this line.

Reply:
I see two typos, would you mind pointing out the third one?

Reply:
I think "move complicatied that" should be "more complicated than".

Reply:
That was the second one, what's the third? https://github.com/rust-lang/rust-analyzer/pull/14154/files

Reply:
1. move -> more
2. complicatied -> complicated
3. that -> than

:thinking: maybe I didn't express myself clearly... your PR seems to fix everything in one go.

Reply:
Yup, I missed `than`, but matklad kindly pointed it out.